_apps/commitlint-bot.md

```diff
@@ -0,0 +1,61 @@
+---
+title: commitlint-bot
+description: A GitHub App that lints all commits of new or edited pull requests and
+  sets an appropriate status check.
+slug: commitlint-bot
+screenshots:
+- https://github.com/ahmed-taj/commitlint-bot/blob/master/docs/status.png
```

Comment:
This should be updated to point to the RAW url: https://github.com/ahmed-taj/commitlint-bot/raw/master/docs/status.png (replaced `blob` with `raw`)